src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -57,7 +59,20 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
       i === 0 && 'first',
       i === breadcrumbs.length - 1 && 'last'
     ),
+    className: classNames(
+      'osdBreadcrumbs',
+      i === 0 && 'first',
+      i > 0 && i < crumbs.length - 1 && 'middle',
+      i === crumbs.length - 1 && 'last'
```

Comment:
Why wouldn't we just use CSS pseudos for these?

I propose storing `crumbs.length` in a variable to avoid having the browser calculate it repeatedly.

Reply:
+1